MIGRATION.md

```diff
@@ -34,6 +34,85 @@ Newman v4 requires Node.js >= v6. [Install Node.js via package manager](https://
 Newman v4 drops support for all the deprecated v2 CLI options, check [V2 to V3 Migration Guide](#v2-to-v3-migration-guide).<br/>
 For the complete list of supported options, see the [README](README.md)
 
+#### --no-color
+This option is dropped in favor of changes made in `color` option. A detailed guide on color option is available below.
```

Comment:
the changes made to the `--color` option. See the section below for more details.